kanidm_client/src/lib.rs

```diff
@@ -571,6 +571,15 @@ impl KanidmClient {
         tokio_block_on(self.asclient.idm_account_purge_attr(id, attr))
     }
 
+    pub fn idm_account_add_attr(
+        &self,
+        id: &str,
+        attr: &str,
+        values: &[&str],
+    ) -> Result<bool, ClientError> {
```

Comment:
It represents past williams mistakes. I thought that there would be a case where you could set and it would be Ok() but change nothing (false), but I think we never indicate that so it should be cleaned up. That could be a new issue from this (cleanup kanidm_client bool/return values).

Reply:
If I wanted to work on this, should I open an issue, create my own branch, and then submit a PR for it? Also, there are a bunch of changes that you made here on this branch, how can I get access to them so I don't have to deal with a lot of merge conflicts?

Reply:
Well, just open the issue first, then once this is merged *then* you create your branch and work from there :)